src/accelerate/accelerator.py

```diff
@@ -1109,6 +1109,20 @@ def prepare(self, *args, device_placement=None):
         >>> # Assume a model, optimizer, data_loader and scheduler are defined
         >>> model, optimizer, data_loader, scheduler = accelerator.prepare(model, optimizer, data_loader, scheduler)
         ```
+
+        ```python
+        >>> from accelerate import Accelerator
+
+        >>> accelerator = Accelerator()
+        >>> # Assume a model, optimizer, data_loader and scheduler are defined
+        >>> device_placement = [True, True, False, False]
+        >>> # Will place the first to items passed in automatically to the right device
+        >>> # The second two will not
```

Comment:
The last sentence is weird here.
```suggestion
        >>> # Will place the first to items passed in automatically to the right device but not the last two.
```